packages/documentation/src/content/docs/integration/playground/overview.mdx

```diff
@@ -52,7 +54,9 @@ To spin up the Local Playground install the following software on your machine:
 
 Run the following commands from the root of the project.
 
-```sh
+<CodeBlock title="Using TigerBeetle">
+
+```
 ## If you have spun up the environment before, first tear down and remove volumes!
 
 pnpm localenv:compose down --volumes
```

Comment:
I actually think we don't need to specify the down anymore, since it should get handled automatically. 
Need to verify this.